addons/hr_holidays/models/hr_leave_accrual_plan.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import datetime
+import logging
+
+from odoo import api, fields, models, _
+from odoo.tools.date_utils import get_quarter, end_of, get_timedelta
+from dateutil.relativedelta import relativedelta
+from datetime import datetime, time
+
+_logger = logging.getLogger(__name__)
+
+#TODO JUD give authorship of this to arj!!!!!!
+
+class AccrualPlan(models.Model):
+    _name = "hr.leave.accrual.plan"
+    _description = "Accrual Plan"
```

Comment:
Script: New model, bootstrap the models + convert the existing accrual into the new mechanism.